### PR TITLE
Update use of FIRST to be consistent across docs

### PR DESCRIPTION
**Pull request by skrueger-ftc**
I noticed that the use of FIRST was not consistent across the site, so I updated them to be. Now, the first instance of *FIRST* in either a heading or on the page includes the registered trademark symbol and all subsequent instances consist of just the name capitalized and italicized. Used [this](https://www.firstinspires.org/sites/default/files/uploads/resource_library/UseofUSFIRSTandLEGOGroupTrademarksandCopyrightedMaterials.pdf) PDF from the branding section on First's website.

### CODE REVIEW

**Comment by abidingabi** [comment]
The FRC Docs team has a lint check specifically for this <https://github.com/wpilibsuite/ohnoyoudidnt/blob/60188f960bf58e60e40846036282c0579003b49a/ohnoyoudidnt/checks.py#L209>, and we use their linter extension. However, I turned that check off specifically because it only works on `FIRST` and `FRC`, but not `FTC`. We should fix that upstream (3 years down the line, oops), and then re-enable those checks. The use of the trademark symbol should probably permeate the entire document, not just the first usage of the word, if we want to match FRC docs (see https://docs.wpilib.org/en/stable/docs/contributing/frc-docs/style-guide.html#text).

**Comment by abidingabi** [comment]
Thank you! We should probably apply the same treatment to `FRC` and `FTC` in regards to the trademark symbol (though not the italicization, see https://docs.wpilib.org/en/stable/docs/contributing/frc-docs/style-guide.html#text).

**Comment by skrueger-ftc** [comment]
Okay. Do you still want me to revert the registered symbol on the first instance only change? I did notice that the \ |reg| on all FIRST instances style was a lot more consistent across the older pages...

**Comment by abidingabi** [comment]
No, I was wrong about that :). The symbol should be only on the first usage according to FIRST’s styleguide. 

It used to be on every instance because for a while we used [ohnoyoudidnt](https://github.com/wpilibsuite/ohnoyoudidnt/tree/main) which enforced that, but we stopped using it at some point (as did FRC docs), as it has some issues and is unmaintained.

**Comment by skrueger-ftc** [comment]
Does this point need to be bolded? Turns out, can't use \ |reg| in a bold block
image

**Comment by abidingabi** [comment]
In regards to text in bold, note that the FRC docs style guide explicitly states that in that situation you do not have to format the trademark:
> Specifically, where possible (i.e. not nested inside other markup or in a document title), the first use of the trademarks should have the ® symbol and all instances of FIRST should be italicized

I think that’s a reasonable thing to follow here as well. Stylistically, I would rather have the whole statement bolded as opposed to having the one word unbolded just for the trademark symbol.

**Comment by skrueger-ftc** [comment]
Oops sorry, cannot read

File: source/docs/contributing/style-guide.rst [hunk 1]
```diff
@@ -41,7 +43,14 @@ Include pictures/videos whenever possible.
 
 Adhere to brand standards when possible.
 
-- It's gm0 not GM0 damn it; look at the logo.
+- Abide by *FIRST*\ |reg| Trademark guidelines, available `here <https://www.firstinspires.org/sites/default/files/uploads/resource_library/UseofUSFIRSTandLEGOGroupTrademarksandCopyrightedMaterials.pdf>`_
+
+  - The first instance of *FIRST* and FTC on a page should include \ |reg| (i.e, *FIRST*\ |reg|)
```
Comment (abidingabi) [inline source/docs/contributing/style-guide.rst:48]:
```suggestion
  - The first instance of *FIRST* and FTC on a page should include |reg| (i.e, *FIRST*\ |reg|)
```

**Comment by abidingabi** [comment]
Heads up, italics don’t work inside of link text, see e.g. https://game-manual-zero--415.org.readthedocs.build/en/415/docs/useful-resources.html#general-resources.
image

File: source/docs/contributing/style-guide.rst [hunk 2]
```diff
@@ -41,7 +43,14 @@ Include pictures/videos whenever possible.
...
 Adhere to brand standards when possible.
 
-- It's gm0 not GM0 damn it; look at the logo.
+- Abide by *FIRST*\ |reg| Trademark guidelines, available `here <https://www.firstinspires.org/sites/default/files/uploads/resource_library/UseofUSFIRSTandLEGOGroupTrademarksandCopyrightedMaterials.pdf>`_
+
+  - The first instance of *FIRST* and FTC on a page should include |reg| (i.e, *FIRST*\ |reg|)
+  - Always capitalize and italize the name *FIRST*
+  - Do NOT use a possessive on the *FIRST* name (e.g, *FIRST*'s)
+
+It's gm0 not GM0 damn it; look at the logo.
```
Comment (abidingabi) [inline source/docs/contributing/style-guide.rst:52]:
```suggestion
- It's gm0 not GM0 damn it; look at the logo.
```

**Pull request merged**